wagtail_automatic_redirects/signal_handlers.py

```diff
@@ -33,8 +30,12 @@ def create_redirect_object_if_slug_changed(sender, **kwargs):
     # JSON and value is stored as JSON in revision.
     page_revisions = instance.revisions.order_by('-created_at', '-id')
     for revision in page_revisions:
-        page_obj = revision.page.specific_class.from_json(
-            revision.content_json)
+        if WAGTAIL_VERSION >= (3, 0):
+            page_obj = revision.page.specific_class.from_json(
```

Comment:
content_json has been renamed from  to content and it now uses [JSONField] in Wagtail 3.0 
https://docs.wagtail.org/en/stable/reference/pages/model_reference.html#wagtail.models.PageRevision.content